fluence-healthcheck/src/index.js

```diff
@@ -0,0 +1,70 @@
+import "@fluencelabs/js-client.node";
+import * as core from "@actions/core";
+import { Fluence } from "@fluencelabs/js-client.api";
+import * as network from "@fluencelabs/fluence-network-environment";
+import { checkPeer } from "./_aqua/main.js";
+
+async function main() {
+  const env = core.getInput("env");
+  const timeout = parseInt(core.getInput("timeout"));
+  let peers, relay;
+
+  const errorMessages = [];
+
+  switch (env) {
+    case "stage":
+      peers = network.stage;
+      relay = network.randomStage();
```

Comment:
why is it random? we need to check all of them

Reply:
randomness in healthchecks is usually a pretty bad idea

Reply:
It checks all of them.
This is just a relay to connect to.
Then the script runs connections checks from each peer to all other peers

Reply:
I understand that. But if a certain relay is down, we will have unpredictable fails.

Reply:
OK I will add a retry to try out a different relay